alfred/response/response.py

```diff
@@ -47,11 +47,12 @@ def _query_response(
         page = int(page)
         total = query.count()
         total_pages = math.ceil(total / per_page)
-
+        next_page = page + 1 if total_pages - page else None
```

Comment:
Sugestão pra deixar mais legível:

```suggestion
        next_page = None
        has_pages = total_pages - page
        if has_pages:
            next_page = page + 1
```

Reply:
vlw pela sugestão girol.